### PR TITLE
(Hopefully) Making playerlist the LitElement way

### PR DESCRIPTION
**Pull request by Moondarker**
Changed according to https://github.com/PrismarineJS/prismarine-web-client/pull/94/files#r593996913

### CODE REVIEW

File: lib/playerlist.js [hunk 1]
```diff
@@ -85,54 +109,9 @@ class PlayerList extends LitElement {
...
-        playerEntry = addPlayer(player)
-      }
-
-      playerEntry.querySelector('.plist-ping-value').innerHTML = player.ping
-    })
-
-    bot.on('playerJoined', addPlayer)
-
-    bot.on('playerLeft', removePlayer)
+    bot.on('playerUpdated', () => this.requestUpdate()) // LitElement seems to be batching requests, so it should be fine?
```
Comment (u9g) [inline lib/playerlist.js:112]: `bot.on('playerUpdated', () => this.requestUpdate())` === `bot.on('playerUpdated', this.requestUpdate)`

Comment (u9g) [reply lib/playerlist.js:112]: this will work for all of the ones below

File: lib/playerlist.js [hunk 2]
```diff
@@ -85,54 +109,9 @@ class PlayerList extends LitElement {
...
+    bot.on('playerJoined', () => this.requestUpdate())
+    bot.on('playerLeft', (player) => () => this.requestUpdate())
```
Comment (u9g) [inline lib/playerlist.js:114]: are you sure this is correct to use double fat arrows? idk seems odd

Comment (Moondarker) [reply lib/playerlist.js:114]: o_O Nope, overlooked that

File: lib/playerlist.js (hunk 1 above)
Comment (Moondarker) [reply lib/playerlist.js:112]: Won't change, it's not ===, try doing that and see what happens to the context (and observe `Uncaught TypeError: this.requestUpdateInternal is not a function` in the console)
We discussed it in #99, and it was mentioned by @extremeheat, the same reason it's good there, it's very bad here

Comment (u9g) [reply lib/playerlist.js:112]: Oh, I see

**Review by u9g — APPROVED** [review]

File: lib/playerlist.js (hunk 1 above)
Comment (extremeheat) [reply lib/playerlist.js:112]: In new JS engines, you can declare arrow functions at class level,
```js
class Class {
  onEvent = (evt) => alert('hello')
}
```

standardjs's parser doesn't support this yet though, so you have to do either bind `this` or do what you did here (only downside here is you can't unregister the event, but not a concern here)

**Review by rom1504 — APPROVED** [review]
Nice
Looks much better

Several other components need the same change if you're interested ;)

**Pull request merged**